### DISCUSSION TITLE
2006/11/09/%E6%B4%97%E6%BE%A1%E7%9A%84%E6%95%85%E4%BA%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#127] [Announcements]
# 2006/11/09/%E6%B4%97%E6%BE%A1%E7%9A%84%E6%95%85%E4%BA%8B

前几天去南京招聘去了，所以也没时间更新 blog。

https://blog.qizhen.xyz/2006/11/09/%E6%B4%97%E6%BE%A1%E7%9A%84%E6%95%85%E4%BA%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：


yoyo
2006年11月17日 18:22 编辑
哈哈

回复
fang
2006年11月13日 17:37 编辑
哈哈，这个男同事不是故意的吧，占mm便宜呢：）

回复
i
2006年11月11日 14:05 编辑
这个…
 
脑筋急转弯
一个女人在河里洗澡,一个男人走过来,她为什么没反应?
(因为他在洗枣)
 
一个女人在河里洗澡,一个男人走过来,那个女人应遮住哪里?(只能是一个部位哦)
 
再来个笑话先
厂长组织员工活动,集体洗浴和参观博物馆.
临出发前,唠叨几句:"今天上午男同志洗澡,女同志参观;下午女同志洗澡,男同志参观.记住了,只准看不准摸,谢绝拍照!"
 
 
第二个问题的答案
(男人的眼睛)

回复
晓元
2006年11月9日 14:27 编辑
好笑话，哈哈
